src/modules/UI/scenes/ChangePinPassword/ChangePasswordComponent.ui.js

```diff
@@ -36,7 +36,7 @@ export default class ChangePassword extends Component<ChangePasswordComponent> {
     return (
       <SafeAreaView>
         <Gradient style={styles.gradient} />
-        <ScrollView style={styles.container}>
+        <ScrollView style={styles.container} keyboardShouldPersistTaps={true}>
```

Comment:
`true` is deprecated. Use `always` instead

https://facebook.github.io/react-native/docs/scrollview#keyboardshouldpersisttaps